### DISCUSSION TITLE
Newbe gets beach ball upon first launch in Pro Tools

### DISCUSSION THREAD

**Discussion by SplashChoke** [apohl79/audiogridder#1018] [Help]
Hi all,

I guess I'm doing something basic wrong. 
Pro Tools 2022.4.0 is running on a MacBook Pro 2017 (macOS 12.2.1), while the AG Server runs on a Mac Mini 2012 (macOS 10.12.6) where some plugins can be found. 

After installation of the server, the search for plugins started automatically. Some of my plugins are available, while quite a few (notably from Waves) failed to load due to their license being unavailable. It would have been nice to be allowed to exclude some locations (or plugin formats) from the search that I knew would fail, but that is a topic for a feature request. 

The server is visible in the tray of the Mac Mini and it seems that the plugins that failed to load show up in the bottom of the Available Plugins list in bold red font (their names are something like: AudioUnit:Effects/aufx,ANSM,ksWV). 

I decided to try out the plugin on the MacBook in a project that had already quite a few plugins loaded locally. Upon inserting the plugin on an AUX track, where I wanted to replace the delay plugin with one hosted on the AG server, I got the beach ball, that wouldn't leave for over an hour.

Any ideas what might have gone wrong?

Kind regards,
Jan

**Comment by apohl79** [comment]
Does not sound like a user error, so something seems broken. I assume the same happens every time? Can you follow the bug reporting (https://audiogridder.com/bug-reports/) guide and provide the plugin and server log/trace files?

**Reply by SplashChoke** [reply]
Thanks for the pointers @apohl79,
I tried to follow the steps in the Bugreport page and noticed that upon restarting the server, a pop-up appeared asking for accessibility privileges. I assumed clicking OK would authorize that, but I checked to be sure and it wasn't set. This may have been my issue in the beginning, but that setting is now OK.

On the Gig Performer [page](https://gigperformer.com/how-to-offload-the-processing-of-audio-plugins-to-remote-computers-using-audiogridder/) describing how to install AG to work with GP, it mentions also permission for Screen Recording is required. This is not available in the Privacy section on my Mac Mini (macOS 10.12.6).

Now, the AG plug-in opens in Pro Tools at 170ms latency, but fails to add any plugins. The error message is: ... EC=E_TIMEOUT STR=unable to retrieve result message after 15 attempts.

**Reply by SplashChoke** [reply]
[AudioGridder_PluginLog.zip](https://github.com/apohl79/audiogridder/files/9542959/AudioGridder_PluginLog.zip)
[AudioGridder_ServerLog.zip](https://github.com/apohl79/audiogridder/files/9542960/AudioGridder_ServerLog.zip)

**Reply by apohl79** [reply]
The server seems to be deadlocking when trying to create a plugin instance. I can't see why as it does not fail. Can you try the latest beta of v1.2?
